package.json

```diff
@@ -19,7 +19,10 @@
     "build": "tsc --project tsconfig.build.json  && npm run assets:copy",
     "start": "npm run build && cd dist && node ./index.js",
     "assets:copy": "copyfiles -f ./config/* ./dist/config && copyfiles -f ./openapi3.yaml ./dist/ && copyfiles ./package.json dist",
-    "clean": "rimraf dist"
+    "clean": "rimraf dist",
+    "docker:build": "docker build -f ./Dockerfile --rm -t vector-tiles-rasterizer:v1 .",
```

Comment:
better if you build it with latest tag `vector-tiles-rasterizer:v1`

Reply:
done.